master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -4,47 +4,88 @@ title: Calico Kubernetes Hosted Install
 
 This document describes deploying Calico on Kubernetes using Kubernetes manifests.  Note that the Kubernetes hosted installation method is experimental and subject to change.
 
-- [`calico.yaml`](calico.yaml): Contains a Kubernetes DaemonSet
-which installs and runs Calico on each Kubernetes master and node. This also includes a ReplicaSet which deploys
-the Calico Kubernetes policy controller, and a ConfigMap which allows for configuration of the install.
+- [`calico.yaml`](calico.yaml): Contains a Kubernetes DaemonSet which installs and runs Calico 
+on each Kubernetes master and node. This also includes a ReplicaSet which deploys the Calico Kubernetes 
+policy controller, and a ConfigMap which allows for configuration of the install.
 
 To install Calico, download [calico.yaml](calico.yaml) and run the following command:
 
 ```shell
 kubectl apply -f calico.yaml
 ```
 
+> **NOTE**
+>
+> Make sure you configure the provided ConfigMap with the location of your etcd cluster before running the above command. 
+
+
 ## How it works
 
 The `calico.yaml` file contains all the necessary resources for installing Calico on each node in your Kubernetes cluster.
 
-It does the following things:
+It installs the following Kubernetes resources: 
 
 - The `calico-config` ConfigMap, which contains parameters for configuring the install.
 - Installs the `calico/node` container on each host using a DaemonSet.
 - Installs the Calico CNI binaries and network config on each host using a DaemonSet.
 - Runs the `calico/kube-policy-controller` pod as a ReplicaSet.
+- The `calico-etcd-secrets` Secret, which optionally allows for providing etcd TLS assets.
 
 ## Configuration options
 
 The ConfigMap in `calico.yaml` provides a way to configure a Calico self-hosted installation.  It exposes
 the following configuration parameters:
 
+## Etcd Configuration
+
+By default, these manifests do not configure secure access to etcd and assume an etcd proxy is running on each host.  The following configuration
+options let you specify custom etcd cluster endpoints as well as TLS.  
+
+To use these manifests with a TLS enabled etcd cluster you must do the following:
+
+- Populate the `calico-etcd-secrets` Secret with the following files: `etcd-ca`, `etcd-key`, `etcd-cert`.
+- Populate the following options in the ConfigMap which will trigger the various services to expect the provided TLS assets: `etcd_ca`, `etcd_key`, `etcd_cert`
+
 ### etcd_endpoints
 
-The location of your etcd cluster.  The default in the provided manifest assumes that an etcd proxy is running on each node.
+A comma separated list of etcd nodes. e.g `https://etcd0:2379,...` 
+
+The default in the provided manifest uses localhost, and assumes that an etcd proxy is running on each node.
+
+### etcd_ca 
```

Comment:
I think this information would be better presented in a table. Up to you if you want to reformat now, or later